practices/perform-static-code-analysis.md

```diff
@@ -1,115 +1,71 @@
+- You are a QA Engineer and you need to identify high risk areas earlier in the development process, so you can identify the best use of your often limited testing time ensuring greater code quality.
+- You are a Tech Lead or Manager and you need to ensure consistent code quality across a team without increasing review overhead.
 
-### Importance of Developer Judgment
+## Goals, Metrics & Signals
 
-While static code analysis tools are helpful, they should not replace developer judgment.
-These tools can highlight potential issues, but it is up to the developers to make the final call on whether a flagged issue is truly problematic.
-Blindly following the tool's recommendations can lead to unnecessary code changes and reduce overall productivity.
-The ability to override automated checks ensures that the development process remains flexible and pragmatic.
+### Intended Outcomes
 
-### Impact on Code Reviews
+- Faster detection and resolution of bugs, security vulnerabilities, and style inconsistencies.
+- Increased developer confidence and reduced rework caused by late-discovered issues.
+- Improved consistency and maintainability across the codebase.
 
-Relying too heavily on static code analysis might lead to a reduction in code reviews.
-Automated tools should complement, not replace, human reviews, which are essential for catching context-specific issues and providing valuable feedback on code design and architecture.
-Ensuring that manual code reviews remain a part of the development process is vital for maintaining high code quality.
+### Target Measurements
 
-## How to Improve
+- Reduced number of issues caught during manual code reviews that could have been flagged by automated tools.
+- Decrease in production bugs linked to preventable coding errors.
+- Positive changes in developer sentiment around “friction” in code reviews, measured via retrospectives or surveys.
 
-### [Do A Spike](/practices/do-a-spike.md)
 
-#### Tool Selection and Initial Setup
+## Lessons From The Field
 
-Identify and set up a static code analysis tool that fits your team's needs.
-Research various static code analysis tools, such as SonarQube or CodeClimate, and compare their features.
-Select one or two tools that seem promising and run them on a small project or segment of your codebase.
-Integrate the chosen tool with your version control system and IDE.
-Review the initial set of issues identified to understand the tool's strengths and weaknesses, and determine which tool aligns best with your workflow.
+- *Automation Should Complement, Not Replace Human Review* – Automated tools are great at spotting patterns but can miss context-specific problems. Keep human judgment in the loop.
+- *False Positives Can Cause Fatigue* – Too many non-issues flagged will erode trust in the tool. Customize rulesets and adjust sensitivity over time.
```

Comment:
There's another concept here coined by Michael Feathers called [review fatigue](https://medium.com/@sageniuz/where-ai-meets-code-techniques-and-best-practices-from-michael-feathers-a-summary-312ef91b6472). Perhaps that's worth incorporating here?